pkg/daemon/ceph/client/mirror.go

```diff
@@ -362,6 +363,39 @@ func ListSnapshotSchedulesRecursively(context *clusterd.Context, clusterInfo *Cl
 	return snapshotSchedulesRecursive, nil
 }
 
+// EnableRBDRadosNamespaceMirroring enables rbd mirroring on a rados namespace.
+func EnableRBDRadosNamespaceMirroring(context *clusterd.Context, clusterInfo *ClusterInfo, poolAndRadosNamespaceName, remoteNamespace, mode string) error {
+	logger.Infof("enable mirroring in rados namespace %s in k8s namespace %q", poolAndRadosNamespaceName, clusterInfo.Namespace)
+	args := []string{"mirror", "pool", "enable", poolAndRadosNamespaceName, mode, "--remote-namespace", remoteNamespace}
+	cmd := NewRBDCommand(context, clusterInfo, args)
+	cmd.JsonOutput = false
+	output, err := cmd.Run()
+	if err != nil {
+		// remove the check when the min supported version is 20.0.0
```

Comment:
Let's check the version when entering this method. If it's a lower version of ceph, we have no reason to run `ceph mirror pool enable`, and we know anyways it will always fail, right?

Reply:
If a user is using a custom ceph image, then it should work.
SO adding the check at top will restrict user to use a custom ceph image

Reply:
Updated as per the discussion